iocBoot/templates/st.cmd.template

```diff
@@ -54,10 +95,12 @@ adsAsynPortDriverConfigure("$(ASYN_PORT)", "$(IPADDR)", "$(AMSID)", "$(AMS_PORT)
 
 cd "$(ADS_IOC_TOP)/db"
 
+{% set motors = get_motors(plc) %}
```

Comment:
Line item in PR description that I want to call out specifically: `Removes motion axes from IOC without pragma (*)`

Related change in pytmc: https://github.com/pcdshub/pytmc/pull/290/files#diff-6be6c30834f696fcba07dcfa6bc5e38c3caacea04cb25945a8f4f59233d55967R329

This requires that motion axes have a pytmc pragma for inclusion into the startup script. 
I believe this would be a good change, but I'd like to confirm it before moving forward here.

Reply:
To be clear: this stops us from creating the placeholder PVs for when someone has left a pragma off of the `DUT_MotionStage` instance? But it should still index the pragma'd motors correctly, skipping over the gaps? I think it's a good change (and I don't think most people will notice)

Reply:
That's correct - indexing and other functionality remains. The records for that axis just do not get created.